docs/url-path-structure.md

```diff
@@ -0,0 +1,20 @@
+# Url path structure
+
+We've agreed on the following structure:
+
+```
+/categories
+/categories?bulkEdit=true
+/categories/:id
+```
+
+* We agreed to omit the `/list` and `/edit` prefixes and go for a more standard
+  approach
+* Bulk editing will be integrated in the list view, so we won’t need a new
+  page/route for this.
```

Comment:
I think this URL scheme is not fully complete. But to what extend it needs to be changed also depends on which views we need. I am mainly unsure about whether or not we'll be implementing a "Item details view". Could you please inform us about that @cooper-joe?

If we _are_ including this view I suggest we use the following URL scheme:

| URL path | Description  |
|--------------|-----------|
| `/categories` | List view |
| `/categories?mode=bulkEdit`      | List view in bulk edit mode* |
| `/categories/add` | Create new item form |
| `/categories/:id` | Item details view |
| `/categories/:id/edit` | Edit item view |

If we _aren't_ including this "item details view", the scheme can be very similar but slightly simpler:

| URL path | Description  |
|--------------|-----------|
| `/categories` | List view |
| `/categories?mode=bulkEdit`      | List view in bulk edit mode* |
| `/categories/add` | Create new item form |
| `/categories/:id` | Edit item view | 

[*] Perhaps the list view will get other modes too, for example we probably want to support inline editing at some point too. Using query parameters for this makes sense, because it is the same view but with another functionality enabled. However, I expect these modes to be mutually exclusive, I don't think a user should be able to have both bulk-editing and inline-editing enabled, so using a single query param with different values probably makes more sense. That's why I changed `?bulkEdit=true` to `?mode=bulkEdit`.